explorer/explorerroutes.go

```diff
@@ -696,18 +709,22 @@ func (exp *explorerUI) ErrorPage(w http.ResponseWriter, code string, message str
 		log.Errorf("Template execute failure: %v", err)
 		str = "Something went very wrong if you can see this, try refreshing"
 	}
+
 	w.Header().Set("Content-Type", "text/html")
-	if notFound {
+	switch sType {
+	case NotFoundStatusType:
 		w.WriteHeader(http.StatusNotFound)
-	} else {
+	case ErrorStatusType:
 		w.WriteHeader(http.StatusInternalServerError)
+	default:
+		w.WriteHeader(http.StatusServiceUnavailable)
 	}
 	io.WriteString(w, str)
 }
 
-// NotFound wraps ErrorPage to display a 404 page
+// NotFound wraps StatusPage to display a 404 page
 func (exp *explorerUI) NotFound(w http.ResponseWriter, r *http.Request) {
-	exp.ErrorPage(w, "Not found", "Cannot find page: "+r.URL.Path, true)
+	exp.StatusPage(w, "Check the URL used.", "Cannot find page: "+r.URL.Path, NotFoundStatusType)
```

Comment:
"Check the URL used." --> "Page not found"

Reply:
Fixed